src/libOpenImageIO/color_ocio.cpp

```diff
@@ -162,9 +162,11 @@ struct CSInfo {
         is_lin_srgb        = 8,   // sRGB/Rec709 primaries, linear response
         is_ACEScg          = 16,  // ACEScg
         is_Rec709          = 32,  // Rec709 primaries and transfer function
-        is_known           = is_srgb | is_lin_srgb | is_ACEScg | is_Rec709
+        is_known           = is_srgb | is_lin_srgb | is_ACEScg | is_Rec709,
+        has_lut3d          = 64,  // has a LUT3D transform
```

Comment:
The has_lut3d flag seems to get set below but never checked?  I think you should be able to remove the Lut3D check once the OCIO bug is fixed, so hopefully there is no future need that would make it useful as part of the CSInfo.

Reply:
Fair enough. Will clean up a bit.

Reply:
Updated